infra/modules/database/backups.tf

```
@@ -4,22 +4,26 @@
 # Backup plan that defines when and how to backup and which backup vault to store backups in
 # See https://docs.aws.amazon.com/aws-backup/latest/devguide/about-backup-plans.html
 resource "aws_backup_plan" "backup_plan" {
+  count = var.enable_aws_backup ? 1 : 0
+
   name = "${var.name}-db-backup-plan"
 
   rule {
     rule_name         = "${var.name}-db-backup-rule"
-    target_vault_name = aws_backup_vault.backup_vault.name
+    target_vault_name = aws_backup_vault.backup_vault[0].name
     schedule          = "cron(0 7 * * ? *)" # Run daily at 7am UTC (2am EST)
 
     lifecycle {
-      delete_after = 21
+      delete_after = 14
```

should this be...

```suggestion
      delete_after = var.backup_retention_period
```

How does this interact with the RDS resource's `backup_retention_period` attribute? 🤔